.github/workflows/tests.yml

```diff
@@ -393,7 +381,7 @@ jobs:
           conda create -n blarg1 -yq python=2.7
```

Comment:
do we still need that python=2.7 since upper and lower bounds are `3.7 and `3.10`

Reply:
since you did change the upper and lower limits in the previous PR.. thought this would be redundant

Reply:
AFAIK this environment is unused in our tests (even if it is used it should be refactored into a pytest fixture) so it can likely be removed entirely, my intention was to do a Windows-only cleanup once we get https://github.com/conda/conda-build/pull/4648 merged